conf/messages.conf

```diff
@@ -289,7 +289,8 @@
 287: You cannot change party leaders in this map.
 //Missing stuff for @killer related commands.
 288: You are no longer killable.
-//289-290 FREE
+289: Target character must be in the same map.
+//290 FREE
```

Comment:
packets.h show at what version packet was added to packets table into client. But after this date for some time packet can be unused or even will be removed and added after. This mean if packet added but unused, it's safe to send this packet to client, but client will ignore it.

Actually packet 0x0291 was added in 2006-10-30, and was not changed and not removed for all the time. (but client may use or not use it for other things)

I am not fixing packets.h for old clients because packets itself can be messy here and with packets.h at same time need fix also clif.c